lib/recommendation.js

```diff
@@ -0,0 +1,61 @@
+var XMLHttpRequest = Cc['@mozilla.org/xmlextras/xmlhttprequest;1'].createInstance();
+
+
+class RecommendationServer {
```

Comment:
I'm using prototypal inheritance elsewhere, let's be consistent and use that for now